python/xorq/ibis_yaml/utils.py

```diff
@@ -117,17 +117,23 @@ def diff_ibis_exprs(expr1, expr2):
     return diffs
 
 
-def translate_storage(storage, compiler: Any) -> Dict:
+def translate_storage(storage, translation_context: Any) -> Dict:
```

Comment:
the user running `xorq run` should be able to re-configure the run on the fly
is that possible here?